swagger.yml

```diff
@@ -0,0 +1,94 @@
+swagger: '2.0'
+info:
+  description: >-
+    The AIOPS Publisher creates a TAR.GZ archive using the raw analyzed data
+    from the AI Service, then invokes the Upload service to upload the archive to S3 and produces a message broadcasting
+    the URL where the data was uploaded
+  version: 1.0.0
+  title: AIOPS Publisher
+  contact:
+    email: support@redhat.com
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+basePath: /
+schemes:
+  - https
+  - http
+paths:
+  /:
+    get:
+      operationId: get.root_path
+      tags:
+      - Liveliness
+      description:
+        Liveliness
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/RootPath'
+      produces:
+      - application/json
+  /api/v1/version:
+    get:
+      operationId: get.version
+      tags:
+      - Version
+      description:
+        Get the current version
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/Version'
+      produces:
+      - application/json
+  /api/v1/invoke_upload_service:
```

Comment:
Since this is the only endpoint this service is providing, why did you choose such a complicated and long name? Why is not `/api/v1/publish` enough?

Reply:
naming is hard sometimes, don't you think so? :)

`publish` seems short and sweet but does not convey the fact that there will be an upload involved in this process.

But either way, I am not too hung up about what the API name should be - so `publish` works for me